pkg/kubeapiserver/options/plugins.go

```diff
@@ -91,6 +92,7 @@ var AllOrderedPlugins = []string{
 	certsubjectrestriction.PluginName,       // CertificateSubjectRestriction
 	defaultingressclass.PluginName,          // DefaultIngressClass
 	denyserviceexternalips.PluginName,       // DenyServiceExternalIPs
+	allowunsafedelete.PluginName,            // AllowUnsafeMalformedObjectDeletion
```

Comment:
Seems like this should be very early in the chain since it fails all the others are not run.

Reply:
I moved it next to `NamespaceAutoProvision`

Reply:
no longer applicable, resolving it